rdrf/rdrf/views/registration_rdrf.py

```diff
@@ -45,7 +45,9 @@ def post(self, request, *args, **kwargs):
         if form.is_valid():
             return self.form_valid(form)
         else:
-            return self.form_invalid(form)
+            context = self.get_context_data(form=form)
```

Comment:
Add logging form validation errors here ( for error in form.errors )

Reply:
done

Reply:
example of log:
runserver_1    | [WARNING:2019-11-04 15:55:46,367:registration_rdrf.py:56:post] Backend validation of the registration has failed: {'username': [{'message': 'Custom user with this Username already exists.', 'code': 'unique'}]}